app/views/stories/show.html.erb

```diff
@@ -137,22 +137,22 @@
 
     <div class="pure-u-1 pure-u-md-6-24">
       <aside class="story-menu">
-        <ul class="plain-list actions-list">
-          <li>
-            <a href="#" class="icon-link link-large">
-              <span class="icon icon-bell"></span>
-              Disable Notifications
-            </a>
-          </li>
-          <% if can? :destroy, @story %>
+        <% if can? :destroy, @story %>
+          <ul class="plain-list actions-list">
+            <!--<li>
```

Comment:
Use Ruby comments where possible and the html will never be sent to the browser:

```
<%#
%>
```